YadavAkhileshh/Alien-Invasion-Defense
# Fix: resolve UI issues

What does this PR do?

This PR addresses several UI issues, including overlapping labels, malfunctioning buttons, and the removal of duplicate code.
Resolved all errors that prevented the game from starting when clicking the 'Start Button' and applied the correct CSS file to improve styling and layout.
These issues were degrading the user interface experience, impacting both functionality and visual clarity. The fixes ensure a more intuitive and responsive UI.
No additional dependencies were required for this change.

Fixes (issue):372

After Changes:
![Screenshot 2024-10-20 231231](https://github.com/user-attachments/assets/087758ff-9646-4566-b7c4-e619b4a70d6d)
![Screenshot 2024-10-20 231734](https://github.com/user-attachments/assets/ade88160-8c75-496a-aae8-e9213f25e6ca)


Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)


How should this be tested?

Click the "Start Button" and confirm that the game starts without any errors. Ensure that this was a consistent issue before and that it has been resolved.
Check the CSS implementation to ensure that styles are applied correctly. Confirm that the correct CSS page is now being loaded and that visual consistency has improved.


Mandatory Tasks
